blazor/datagrid/cell-edit-types.md

```diff
@@ -649,6 +649,75 @@ You can able to render SfTimePicker component in EditTemplate. In the below samp
 In the following image, **SfTimePicker** component is rendered with **EditTemplate** in OrderDate column
 ![Blazor DataGrid with Editing in Custom TimePicker](./images/blazor-datagrid-editing-in-custom-timepicker.png)
 
+### Using TextBox in EditTemplate
+
+By defining the [EditTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_EditTemplate) feature of a [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html), you can render a custom editor component in the Grid edit form.
```

Comment:
BY default enter key in grid will save the edit changes. 
 
But mulitiline text box 

enter key will insert a new line and type new text . At office pressing The enter key in multiline text box caused to save Changes instead of inter ab blank new ro